101-webapp-linux-sonarqube-mysql/azuredeploy.json

```diff
@@ -0,0 +1,218 @@
+      }
+    },
+    "databaseSkuTier": {
+      "type": "string",
+      "allowedValues": [
+        "Basic"
+      ],
+      "defaultValue": "Basic",
+      "metadata": {
+        "description": "Azure database for MySQL pricing tier"
+      }
+    },
+    "mysqlVersion": {
+      "type": "string",
+      "allowedValues": [
+        "5.6",
+        "5.7"
+      ],
+      "defaultValue": "5.7",
+      "metadata": {
+        "description": "MySQL version"
+      }
+    }
+  },
+  "variables": {
+    "databaseName": "[concat(parameters('siteName'), 'database')]",
+    "serverName": "[concat(parameters('siteName'), 'pgserver')]",
+    "jdbcSonarUserName": "[concat(parameters('administratorLogin'),'@',variables('serverName'))]",
+    "hostingPlanName": "[concat(parameters('siteName'), 'serviceplan')]"
+
+  },
+  "resources": [
+    {
+      "comments":"This is the Linux web app with the Sonarquebe Docker image and the DB",
+      "type": "Microsoft.Web/sites",
+      "name": "[parameters('siteName')]",
+      "properties": {
+        "siteConfig": {
+          "linuxFxVersion": "DOCKER|SONARQUBE"
+        },
+        "name": "[parameters('siteName')]",
+        "serverFarmId": "[variables('hostingPlanName')]",
+        "hostingEnvironment": ""
+      },
+      "apiVersion": "2016-03-01",
+      "location": "[resourceGroup().location]",
+      "dependsOn": [
+        "[variables('hostingPlanName')]",
+        "[variables('databaseName')]"
+      ],
+      "resources": [
+        {
+          "name": "appsettings",
+          "type": "config",
+          "apiVersion": "2015-08-01",
+          "comments": "This are the generated settings for the Sonar jdbc connectionstring",
+          "dependsOn": [
+            "[resourceId('Microsoft.Web/sites', parameters('siteName'))]"
+          ],
+          "tags": {
+            "displayName": "SonarappSettings"
+          },
+          "properties": {
+            "SONARQUBE_JDBC_URL": "[concat('jdbc:mysql://', variables('serverName') ,'.mysql.database.azure.com:3306/',variables('databaseName'),'?verifyServerCertificate=true&useSSL=true&requireSSL=false&useUnicode=true&characterEncoding=utf8')]",
+            "SONARQUBE_JDBC_USERNAME": "[variables('jdbcSonarUserName')]",
+            "SONARQUBE_JDBC_PASSWORD": "[parameters('administratorLoginPassword')]"
+          }
+        }
+      ]
+    },
+    {
+      "apiVersion": "2016-09-01",
+      "name": "[variables('hostingPlanName')]",
+      "type": "Microsoft.Web/serverfarms",
+      "location": "[resourceGroup().location]",
+      "properties": {
+        "name": "[variables('hostingPlanName')]",
+        "workerSizeId": "1",
+        "reserved": true,
+        "numberOfWorkers": "1",
+        "hostingEnvironment": ""
+      },
+      "sku": {
+        "Tier": "[parameters('servicePlanTier')]",
+        "Name": "[parameters('servicePlanSku')]"
+      },
+      "kind": "linux"
+    },
+    {
+      "apiVersion": "2016-02-01-privatepreview",
```

Comment:
is there a later/non-private apiVersion available

Reply:
Solved in next PR